connectors/airwatch/src/test/java/com/vmware/connectors/airwatch/AirWatchControllerTest.java

```diff
@@ -107,12 +107,20 @@ void testDiscovery() throws IOException {
                 .exchange()
                 .expectStatus().is2xxSuccessful()
                 .expectBody().json(fromFile("/connector/responses/discovery.json"));
+
+        String expectedMetadata = fromFile("/connector/responses/metadata.json");
+        String localHostRegex = "(http|https):\\/\\/localhost:\\d{4,5}";
         webClient.get()
                 .uri("/discovery/metadata.json")
                 .headers(ControllerTestsBase::headers)
                 .exchange()
                 .expectStatus().is2xxSuccessful()
-                .expectBody().json(fromFile("/connector/responses/metadata.json"));
+                .expectBody()
+                .consumeWith(res -> {
+                    String body = new String(res.getResponseBody());
+                    assertThat(body.replaceAll(localHostRegex, "CONNECTOR_HOST"),
```

Comment:
Instead of ```CONNECTOR_HOST```, we can use ```http://localhost```.

Reply:
The objective of the change is to replace ```CONNECTOR_HOST``` with the server URL in an actual scenario. It will be confusing when we change ```CONNECTOR_HOST``` again with ```CONNECTOR_HOST```. It would be better if we change ```CONNECTOR_HOST``` with some static URL like ```http://localhost``` and check with the response.

Reply:
Yes, readability can be improved in Airwatch connector UT. It is having a separate metadata JSON file for unit tests. 
For others, I still use the same static metadata file, which is used by the connector controller. (In common ControllerTestsBase)

Done.